tests/sentry/hybrid_cloud/test_actor.py

```diff
@@ -46,8 +42,6 @@ def test_many_from_object_users_missing_actors():
 
     actors = RpcActor.many_from_object(users)
     assert len(actors) == len(users)
-    assert actors[0].actor_id
-    assert actors[1].actor_id
 
     actors = Actor.objects.filter(type=ACTOR_TYPES["user"])
     assert len(actors) == 2, "Actors should be generated"
```

Comment:
Should this part of the test be removed as we're not generating new actors anymore because `get_actor_for_user` is no longer called.